docs/workflow/testing/mono/testing.md

```diff
@@ -1,54 +1,114 @@
-# Running Tests using Mono Runtime
+# Running test suites using Mono
 
-## Running Runtime Tests
-We currently only support running tests against coreclr.  There are additional mono runtime tests in mono/mono, but they
-have not been moved over yet. Simply run the following command:
+Before running tests, [build Mono](../../building/mono/README.md) using the desired configuration.
 
+## Runtime Tests
+### Testing against Mono JIT:
+
+* Build runtime tests by executing the follwiing command from `$(REPO_ROOT)/src/tests`
+```
+./build.sh excludemonofailures <release|debug>
+```
+
+* Running a single test:
+```
+cd ../mono/netcore
```

Comment:
Again, I don't think you want to put this cd twice. Just say "from `$(REPO_ROOT)/src/tests`" or something.

Reply:
The document was written from a developer point of view. Personally, I prefer to type less when possible. That's why I put the command there directly for people to copy and paste. I don't think this matters much. However, I could see from your perspective, it is a duplication.